docs/docusaurus/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_identify_your_data_context_expectations_store.mdx

```diff
@@ -1,6 +1,6 @@
 import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
 
-You can add a new <TechnicalTag tag="expectation_store" text="Expectation Store" /> to your <TechnicalTag tag="data_context" text="Data Context" /> by using the `add_store()` metho by using the `add_store()` methodd.
+You can add a new <TechnicalTag tag="expectation_store" text="Expectation Store" /> to your <TechnicalTag tag="data_context" text="Data Context" /> by using the `add_store()` metho by using the `add_store()` method.
```

Comment:
```suggestion
You can add a new <TechnicalTag tag="expectation_store" text="Expectation Store" /> to your <TechnicalTag tag="data_context" text="Data Context" /> by using the `add_store()` method.
```

Reply:
Also this line feels out of place, the section heading is `1.4.1 Identify your Data Context Expectations Store` not to add a store. Could this instead be consistent with the text in the validations store section: https://deploy-preview-7868--niobium-lead-7998.netlify.app/docs/deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_redshift#151-identify-your-data-contexts-validation-results-store
```
You can find your [Validation Results Store](https://deploy-preview-7868--niobium-lead-7998.netlify.app/docs/terms/validation_result_store)A connector to store and retrieve information about objects generated when data is Validated against an Expectation Suite. configuration within your [Data Context](https://deploy-preview-7868--niobium-lead-7998.netlify.app/docs/terms/data_context).

Look for the following section in your [Data Context's](https://deploy-preview-7868--niobium-lead-7998.netlify.app/docs/terms/data_context) great_expectations.yml file:
```